src/index.ts

```diff
@@ -488,20 +541,26 @@ export class Bigtable {
     const adminOptions = Object.assign(
       {},
       baseOptions,
+      getUniverseDomainOptions(options, options.BigtableTableAdminClient),
```

Comment:
Without this, ie. without providing a universeDomain to the gapic client we see an error that says: `Error: The configured universe domain (googleapis.com) does not match the universe domain found in the credentials (apis-tpczero.goog). If you haven't configured the universe domain explicitly, googleapis.com is the default.`